website/src/content/docs/docs/libraries/http/reference/data-types.md

```diff
@@ -218,16 +218,22 @@ model TypeSpec.Http.CreatedResponse
 ### `File` {#TypeSpec.Http.File}
 
 ```typespec
-model TypeSpec.Http.File
+model TypeSpec.Http.File<Contents>
 ```
 
+#### Template Parameters
+
+| Name     | Description |
+| -------- | ----------- |
+| Contents |             |
+
 #### Properties
 
-| Name         | Type     | Description |
-| ------------ | -------- | ----------- |
-| contentType? | `string` |             |
-| filename?    | `string` |             |
-| contents     | `bytes`  |             |
+| Name         | Type       | Description |
+| ------------ | ---------- | ----------- |
+| contentType? | `string`   |             |
+| filename?    | `string`   |             |
+| contents     | `Contents` |             |
```

Comment:
Could you add description how to decide to use `string` or `bytes`
And describe `Contents` can be one of `string` or `bytes` or their derived scalar?

Reply:
Yes, as the checklist in the PR description says, the documentation here is a work in progress so I will be sure to fill this out with details about how to use these arguments and also add extensive documentation to the TypeSpec website that indicates how spec authors use Http.File.